Content.Shared/Chemistry/Components/SharedChemMasterComponent.cs

```diff
@@ -70,10 +72,11 @@ public class UiActionMessage : BoundUserInterfaceMessage
             public readonly FixedPoint2 amount;
             public readonly string id = "";
             public readonly bool isBuffer;
+            public readonly FixedPoint2 pillType;
             public readonly int pillAmount;
             public readonly int bottleAmount;
 
-            public UiActionMessage(UiAction _action, FixedPoint2? _amount, string? _id, bool? _isBuffer, int? _pillAmount, int? _bottleAmount)
+            public UiActionMessage(UiAction _action, FixedPoint2? _amount, string? _id, bool? _isBuffer, FixedPoint2? _pillType, int? _pillAmount, int? _bottleAmount)
```

Comment:
It wasn't you who did this, but none of these follow the normal variable naming conventions. 

Public variables should be PascalCase, local variables (like function args) shouldn't have underscores. You didn't cause this, but it might as well be fixed in this PR.